### PR TITLE
Fix markdown preview focus

### LINKED ISSUE #543: Markdown preview steals focus
- `code-server` version: 1.868-vsc1.33.1
- OS Version: Ubuntu 18.04

## Description

Preview window of markdown steals focus while editing the corresponding markdown file.

## Steps to Reproduce

1. Open preview window to the side.
1. Edit the markdown file.
    For example, try to add following texts:
    ```markdown
    ### abc
    ```
    As the preview window steals focus, you will not be able to write `abc`.

### PR DESCRIPTION
**Pull request by nol166**
### Describe in detail the problem you had and how this PR fixes it
Fixes markdown preview from stealing focus away from the editor while typing

### Is there an open issue you can link to?
fixes #543

### CODE REVIEW

File: scripts/vscode.patch [hunk 1]
```diff
@@ -824,6 +824,15 @@ index 74fc798..03728d0 100644
 +			// 	supportFetchAPI: true,
 +			// 	corsEnabled: true
 +			// });
+@@ -310 +328 @@
+-
++			
+@@ -328 +346,4 @@
+-					newFrame.contentWindow.focus();
++					/* Comment out to avoid VS Code from focusing on markdown preview on reload */
```
Comment (code-asher) [inline scripts/vscode.patch]: This comment doesn't match the behavior; maybe something like "Only focus the updated frame if it already had focus"?

File: scripts/vscode.patch [hunk 2]
```diff
@@ -885,7 +894,7 @@ index 48ef482..dc47f81 100644
 -			placeHolder: isMacintosh ? nls.localize('openRecentPlaceHolderMac', "Select to open (hold Cmd-key to open in new window)") : nls.localize('openRecentPlaceHolder', "Select to open (hold Ctrl-key to open in new window)"),
 +			placeHolder: browser.isMacintosh ? nls.localize('openRecentPlaceHolderMac', "Select to open (hold Cmd-key to open in new window)") : nls.localize('openRecentPlaceHolder', "Select to open (hold Ctrl-key to open in new window)"),
 diff --git a/src/vs/workbench/electron-browser/main.contribution.ts b/src/vs/workbench/electron-browser/main.contribution.ts
-index 71bc992..111ce51 100644
+index 71bc992..2692f39 100644
```
Comment (code-asher) [inline scripts/vscode.patch]: Looks like there's a conflict here with master that needs to be resolved. You could just revert this line, or merge and regenerate the patch. I'm not sure why it generated a different hash. :confused:

File: scripts/vscode.patch (hunk 1 above)
Comment (nol166) [reply scripts/vscode.patch]: fixed

File: scripts/vscode.patch (hunk 2 above)
Comment (nol166) [reply scripts/vscode.patch]: fixed

File: scripts/vscode.patch [hunk 3]
```diff
@@ -824,6 +824,16 @@ index 74fc798..03728d0 100644
 +			// 	supportFetchAPI: true,
 +			// 	corsEnabled: true
 +			// });
+@@ -310 +328 @@
+-
```
Comment (code-asher) [inline scripts/vscode.patch]: We might want to remove all the extra whitespace here so the patch is as small as possible.

**Pull request merged**